docs/docs/03-language-reference.md

```diff
@@ -1771,6 +1771,24 @@ bring cloud; // from cloud bring * as cloud;
 bring "cdktf" as cdktf; // from "cdktf" bring * as cdktf;
 ```
 
+To import an individual Wing file as a module, you can specify its path relative
+to the current file:
+
+```TS
+bring "./my-module.w" as myModule;
+```
+
+It's also possible to import a directory as a module. The module will contain all
+types defined in the directory's files. If the directory has subdirectories,
```

Comment:
This should eventually be "all public types", correct?

I am 100% okay with this not included in the current PR (and I understand why), but definitely worth adding a "Future items" section to the PR description indicating this is the intention.

Reply:
Yep you're right - will update